hnn_core/parallel_backends.py

```diff
@@ -115,6 +115,8 @@ def simulate(self, net):
 
         parallel, myfunc = self._parallel_func(self._clone_and_simulate)
         sim_data = parallel(myfunc(net, idx) for idx in range(n_trials))
+        # XXX: why don't we do:
+        # dpls, spike_data = parallel(...)
```

Comment:
Sounds reasonable, esp. since `dpls` doens't need to be passed to `_gather_trial_data()`. We will want to add a third element for somatic voltages once that functionality is added to back to hnn-core from HNN.

Another option would be a dictionary `sim_data['dpls']`, for ease of adding more types in the future.

Reply:
I just find it better for my sanity to use fewer variable names. We can make it a dictionary in the future if the need arises :-)